templates/addons/windows/calico/kube-proxy-windows.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-proxy-windows
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-proxy-windows
+    spec:
+      serviceAccountName: kube-proxy
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      containers:
+      - image: sigwindowstools/kube-proxy:${KUBERNETES_VERSION/+/_}-calico-hostprocess
```

Comment:
is this a special kube-proxy image?

Reply:
Yes, it re-uses the scripts provided by kubeproxy (modified for this hostprocess) see more details at https://github.com/kubernetes-sigs/sig-windows-tools/pull/161/

Reply:
eventually I would like to get the kubeproxy automatically deployed by kubeadm as well as make it configurable through kubeadm

Reply:
interesting! thanks for the link to the other PR